components/Newsfeed/Newsfeed.tsx

```diff
@@ -175,46 +165,50 @@ export default function Newsfeed() {
                               bodyImgAltTxt={``}
                               bodyText={element.bodyText}
                               isBillMatch={element.isBillMatch}
+                              isUserMatch={element.isUserMatch}
+                              type={element.type}
                             />
                           ) : (
-                            <AlertCard
+                            <NewsfeedCard
+                              court={element.court}
                               header={element.header}
                               subheader={element.subheader}
                               timestamp={element.timestamp}
-                              headerImgSrc={`${
-                                element.type === `bill`
-                                  ? ``
-                                  : `/thumbs-${element.position}.svg`
-                              }`}
-                              headerImgTitle={`${
-                                element.type === `bill` ? "" : element.position
-                              }`}
-                              bodyImgSrc={``}
-                              bodyImgAltTxt={``}
+                              headerImgSrc={`/individualUser.svg`}
```

Comment:
nit: Rather than setting the img/alt text here, could we put the responsibilty for choosing those to the `NewsfeedCard` body subcomponents? `NewsfeedCard` is already switching on type - we may as well handle all of the type-specific formatting in one place (and then we could also ditch the conditional and just render all types of notification here as `NewsfeedCard` and differentiate between different body types in the `NewsfeedCard` component itself).

Reply:
yes, done